test/intercept/undici/undici.fetch.test.ts

```diff
@@ -0,0 +1,190 @@
+/**
```

Comment:
We can put these test suites into `test/integration` folder. It's there to test this library's integration with other libraries (like `got`), as those may have certain corner-cases to account for.

Reply:
As I see it, `undici` is a replacement for the old `http` and `https` module, and many libraries [include got](https://github.com/sindresorhus/got/issues/1419) may use it instead of `http` in the future. So, I suppose this test should be placed nearby the tests of the `http` interceptor.

Reply:
@chentsulin, well, `undici` is still a _third-party library_. Until it's a part of the Node.js standard library, we should treat it as third-party integration.

Reply:
Fair enough. We can rethink about that by then.